pkg/storage/rdbms/test_suite.go

```diff
@@ -14,12 +14,12 @@ type TestSuite struct {
 // NewTestSuite creates a common test suite
 func NewTestSuite(model *suite.Model) (storage.TestSuite, error) {
 	conn := model.DB
-	rdbmsStore := NewRDBMSStorageWithConn(conn)
+	rdbmsStore, err := NewRDBMSStorageWithConn(conn)
 
```

Comment:
can you check the error here and return it with a nil TestSuite instead? Then nil on last return.